src/HonzaBotner.Discord/Command/Permissions/Entities/PermissionEntry.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace HonzaBotner.Discord.Command.Permissions.Entities
+{
+    public class PermissionEntry
```

Comment:
Is it good idea to have database object in .Discord project? I don't think so. You should probably expect here some Dto like object, which doesn't have any direct reference to the database.